### DISCUSSION TITLE
Dark-mode

### DISCUSSION THREAD

**Discussion by piotrek124-1** [electricitymaps/electricitymaps-contrib#3325] [Ideas] [answered]
Hey,
The current dark-mode is quite bright at night. What do you think about also changing such elements as icons, header etc, To look more like that:
![Zrzut ekranu z 2021-08-18 22-55-20](https://user-images.githubusercontent.com/66417442/130243941-00b9aec1-e439-4abc-ab07-9a09a93908eb.png)

**Comment by madsnedergaard** [comment]
That would be awesome! I actually started working on this, but never got around to wrap it up - the branch is here if you or anyone wants to continue on it: https://github.com/tmrowco/electricitymap-contrib/tree/mn/dark-mode :)

**Comment by madsnedergaard** [comment, marked as answer]
This has now been done as part of #4731 :)
